### DISCUSSION TITLE
Duplicate arrows when using `navigation` with Swiper Element

### DISCUSSION THREAD

**Discussion by darbymanning** [nolimits4web/swiper#6777] [Q&A]
I wasn't sure if this was a bug, so thought I'd post this here first. But when using the `navigation` prop with custom `nextEl` and `prevEl` selectors, I find that Swiper also adds its own buttons for prev & next slides. This might be intended behaviour, but I'm always using this prop to _replace_ the elements Swiper generates, so I can have more control of the markup.

I've also found that this doesn't occur if the instance of swiper gets created in JavaScript (ie. using `init="false"` on the custom element, and providing all the arguments for Swiper when instantiating it. This makes me feel like this is probably a bug.

The fix I've gone with for now is simply to change the styling of the provided buttons so they aren't visible. Here's a little codesandbox with the issue replicated:

https://codesandbox.io/s/swiper-duplicate-navigation-434rlm?file=/index.html

.. I guess to make this a question, is this a bug? ;)

**Comment by nolimits4web** [comment]
The demo you have posted doesn't illustrate the issue

**Reply by darbymanning** [reply]
Sorry, you're right. I think the actual problem here is that you cannot provide a JS objects as values for props when using Swiper Element. Swiper is simply seeing `navigation` as truthy as a result - so that's why my custom buttons don't work here.

I guess this is why demos tend to instantiate Swiper manually on JavaScript if they are doing this, right? Would you consider parsing JSON provided as props in future?
